src/components/modal/CdrModal.vue

```diff
@@ -190,6 +201,62 @@ const removeNoScroll = () => {
   }
 };
 
+const backgroundContentNodes: Array<HTMLElement> = [];
+let backgroundNodesDiscovered = false;
+
+/**
+ * findBackgroundContentNodes
+ * When modal opens for the first time, find the content that should
+ * be hidden when it opens, and store it in an array
+ */
+const findBackgroundContentNodes = () => {
+  // initial selector is complex, breaking it down
+  const notSelectors = [
+    'body > *',
+    ':not(script)',
+    ':not(style)',
+    ':not([data-is-modal])',
+    ':not([aria-hidden=true])',
+  ]
+  const contentBodyChildren: NodeListOf<HTMLElement> =
+    document.querySelectorAll(notSelectors.join(''));
+
+  // for the remaining, check if they should be hidden
+  contentBodyChildren.forEach((el) => {
+    if (getComputedStyle(el).getPropertyValue('display') !== 'none') {
+      // TODO - temp to be sure we tagged right
+      el.setAttribute('data-modal-hide', '');
```

Comment:
This isn't strictly necessary, but would be pretty helpful in debugging.  I'm 50/50 on whether to leave it or remove it.